net/nginx-util/Makefile

```diff
@@ -15,27 +15,27 @@ define Package/nginx-util/default
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=Web Servers/Proxies
-  TITLE:=Builder of LAN listen directives for Nginx
+  TITLE:=Nginx configurator
   DEPENDS:=+libstdcpp +libubus +libubox +libpthread
 endef
 
 define Package/nginx-util
   $(Package/nginx-util/default)
-  CONFLICTS:=nginx-ssl-util
+  CONFLICTS:=nginx-ssl-util-nopcre nginx-ssl-util
 endef
 
 define Package/nginx-ssl-util/default
   $(Package/nginx-util/default)
-  TITLE+= and manager of its SSL certificates
+  TITLE+= including SSL
   DEPENDS+= +libopenssl
-  CONFLICTS:=nginx-util
+  CONFLICTS:=nginx-util,
 endef
 
 define Package/nginx-ssl-util
   $(Package/nginx-ssl-util/default)
```

Comment:
This looks like a bug. Note that /default has a conflicts line. And so does this.